src/xclim/indices/_conversion.py

```diff
@@ -1319,6 +1319,58 @@ def _get_D_from_M(time):  # noqa: N802
     )
 
 
+# TODO: How can I check exact units for inputs
+# @declare_units(
+#     net_radiation="[MJ m-2 day-1]",
+#     tas="[degC]",
+#     wind="[m s-1]",
+#     es="[kPa]",
+#     ea="[kPa]",
+#     delta_svp="[kPa degC-1]",
+#     gamma="[kPa degC]",
+#     G="[MJ m-2 day-1]",
+# )
```

Comment:
Xclim's way would be to prescribe dimensionality and to explicitly convert every input in the beginning of the function, like:
```python3
tas = convert_units_to(tas, '°C')
```

Reply:
Alright i guess thats actually more user-friendly. 

How do I declare units like kPa * degC though?